samples/Calling/src/app/App.tsx

```diff
@@ -56,6 +56,8 @@ const App = (): JSX.Element => {
   /* @conditional-compile-remove(rooms) */
   const [role, setRole] = useState<Role>();
 
+  const [isTeamsCall, setIsTeamsCall] = useState<boolean>(false);
```

Comment:
nit: almost worth conditionally compiling this line just so when we update the public samples it obvious whether or not to keep this line